### PR TITLE
Add Django-Watchman check for Elasticsearch health

### PR DESCRIPTION
**Pull request by willbarton**
This PR adds a check for Elasticsearch cluster health to our Django-Watchman configuration. It should report "ok" unless the Elasticsearch API reports a time-out or if the cluster status isn't "green". It includes the ES health API output at the shard-level.

This PR also removes the Django-Watchman configuration we have that we haven't been using. It also updates the default Watchman token generation to use the Python 3 `secrets` library.

Locally, [`WATCHMAN_TOKENS` is empty](https://github.com/cfpb/consumerfinance.gov/blob/main/.env_SAMPLE#L176) so Watchman should be available http://localhost:8000/_status/

## How to test this PR

1. Visit http://localhost:8000/_status/ and observe the Elasticsearch check output.

## Checklist

- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)

### CODE REVIEW

File: cfgov/cfgov/settings/base.py [hunk 1]
```diff
@@ -765,15 +766,12 @@
...
 # Watchman tokens, used to authenticate global status endpoint
-WATCHMAN_TOKENS = os.environ.get("WATCHMAN_TOKENS", os.urandom(32))
+WATCHMAN_TOKENS = os.environ.get("WATCHMAN_TOKENS", secrets.token_hex())
 
 # This specifies what checks Watchman should run and include in its output
 # https://github.com/mwarkentin/django-watchman#custom-checks
 WATCHMAN_CHECKS = (
-    "watchman.checks.databases",
-    "watchman.checks.storage",
-    "watchman.checks.caches",
```
[the remark this answers is not in the document]
Comment (willbarton) [reply cfgov/cfgov/settings/base.py:775]: They've been there for years and we haven't used them. I'm trying to pare down the information to what we actually might need.

[the remark this answers is not in the document]
Comment (willbarton) [reply cfgov/cfgov/settings/base.py:775]: Then I don't think they've ever been useful :)

[the remark this answers is not in the document]
Comment (Scotchester) [reply cfgov/cfgov/settings/base.py:775]: I know you know that logic is flawed, but it's not a hill I'm going to die on 😝

**Review by Scotchester — APPROVED** [review]
Works as advertised. Any particular reason this is still in draft?

**Comment by willbarton** [comment]
> Works as advertised. Any particular reason this is still in draft?

Mostly just wasn't sure if this is something we want to do.

File: cfgov/cfgov/settings/base.py [hunk 2]
```diff
@@ -765,15 +766,12 @@
...
-    "alerts.checks.check_clock_drift",
+    "alerts.checks.elasticsearch_health",
 )
 
 # Used to check server's time against in check_clock_drift
```
[the remark this answers is not in the document]
Comment (willbarton) [reply cfgov/cfgov/settings/base.py]: Done!

**Pull request merged**